presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestAggregations.java

```diff
@@ -858,6 +858,24 @@ public void testGroupByNanMap()
         assertTrue(Double.isNaN((Double) actual.getOnlyValue()));
     }
 
+    @Test
+    public void testMinMaxOfNaN()
```

Comment:
Did you find the place where other double/float aggregations are tested?
Would it be possible to place these test cases there?

Also, this looks like testing doubles only. What about `REAL` (32-bit float)?